dm/pkg/checker/mysql_server.go

```diff
@@ -35,14 +35,14 @@ func NewMySQLVersionChecker(db *sql.DB, dbinfo *dbutil.DBConfig) RealChecker {
 }
 
 // SupportedVersion defines the MySQL/MariaDB version that DM/syncer supports
-// * 5.6.0 <= MySQL Version < 8.0.0.
+// * 5.6.0 <= MySQL Version <= 8.0.24.
```

Comment:
we should also report error when found PARTIAL_UPDATE_ROWS_EVENT when increase the supported version

https://github.com/go-mysql-org/go-mysql/pull/774

Reply:
Hi, kindly remind that go-mysql has merged some commit about MySQL 8.0 features, please remember to update the library

Reply:
Done in d21bf42cb748f68de7721039ce09f9e45ef5c0af